ali-dbhub-server/pom.xml

```diff
@@ -107,7 +107,7 @@
             <dependency>
                 <groupId>cn.hutool</groupId>
                 <artifactId>hutool-all</artifactId>
-                <version>5.8.5</version>
+                <version>5.8.18</version>
```

Comment:
For PRs that modify documentation, dependency version numbers should not be changed. If a change in dependency version number is required, a separate PR should be opened for it and tests need to be conducted.